### DISCUSSION TITLE
Poosible to run PlayStation OS in UTM?

### DISCUSSION THREAD

**Discussion by LostInCompilation** [utmapp/UTM#4912] [UTM Help (macOS)]
I just discovered the PlayStation and PSP logos in UTM. Is it possible to run these OSs in UTM? What would be the VM configuration and which PlayStation System is supported (2, 3, 4, 5)? And where to get those from? AFAIK PS2 doesn't have a OS.

**Comment by osy** [comment]
Uh no

**Comment by LostInCompilation** [comment]
Why are the PS/PSP logos in UTM in the first place?

**Reply by osy** [reply]
It’s from a pack of OS logos.
